plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -699,17 +698,6 @@ public boolean configure(final String name, final Map<String, Object> params) th
             _hypervisorType = HypervisorType.KVM;
         }
 
-        //Verify that cpu,cpuacct cgroups are not co-mounted
-        if(HypervisorType.LXC.equals(getHypervisorType())){
```

Comment:
@kishankavala can you review this patch, it removes some of the cgroup setting stuff for LXC?